src/SolutionTemplate/UnoSolutionTemplate.VISX/UnoPlatformPackage.cs

```diff
@@ -148,16 +157,30 @@ private void SetupOutputWindow()
 			_infoAction = s => pane().OutputString("[INFO] " + s + "\r\n");
 			_verboseAction = s => pane().OutputString("[VERBOSE] " + s + "\r\n");
 			_warningAction = s => pane().OutputString("[WARNING] " + s + "\r\n");
-			_errorAction = e => pane().OutputString("[ERROR] " + e.ToString() + "\r\n");
+			_errorAction = e =>
+			{
+				if (System.Diagnostics.Debugger.IsAttached)
+				{
+					pane().OutputString("[ERROR] " + e.ToString() + "\r\n");
```

Comment:
Codacy found an issue: [There's no need to call 'ToString()', the compiler will do it for you.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6153895)